en/docs/services/create-your-first-service.md

```diff
@@ -37,15 +37,15 @@ To create the service, follow the procedure below:
         
     3. Click the **+** icon below the HTTP API call you added. 
     
-        ![Update Existing Connection](../assets/img/services/update-existing-connection.png)
+        ![Update Existing Connection](../assets/img/services/update-existing-connection.png){.cInlineImage-half}
         
     4. Click **API Calls**, and then click on the existing connection.
     
-        ![Select Existing connection](../assets/img/services/select-existing-connection.png)
+        ![Select Existing connection](../assets/img/services/select-existing-connection.png){.cInlineImage-half}
         
     5. Enter information as follows:
         
-        ![Configure HTTP Connection](../assets/img/services/select-existing-connection.png)
+        ![Configure HTTP Connection](../assets/img/services/select-existing-connection.png){.cInlineImage-half}
```

Comment:
same here

Reply:
Image size is not visible in github view mode. This shows the original image size.